test/failed-auth.test.js

```diff
@@ -3,7 +3,7 @@
 const test = require('tap').test
 const anger = require('..')
 
-test('failed auth', { timeout: 3000 }, (t) => {
+test('failed auth', { timeout: 20000 }, (t) => {
```

Comment:
I think this test should be quick, and you need to add a separate test on the backoff, checking that it is, in fact, retrying.